pkg/controllers/streaming/processor_controller.go

```diff
@@ -532,6 +533,15 @@ func (r *ProcessorReconciler) constructDeploymentForProcessor(processor *streami
 		ImagePullPolicy: v1.PullIfNotPresent,
 		Env:             environmentVariables,
 		VolumeMounts:    volumeMounts,
+		ReadinessProbe: &v1.Probe{
+			Handler: v1.Handler{
+				TCPSocket: &v1.TCPSocketAction{
+					Port: intstr.IntOrString{
```

Comment:
favor the helper methods, like `intstr.FromInt()`

Reply:
TIL!